CHANGELOG.md

```diff
@@ -22,12 +27,28 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 ...
 
-## 0.1.1 (2024-05-27)
+## [0.1.1][] (2024-05-27)
+
+This release has an [MSRV] of 1.70.
 
 ### Added
 
-- [#26](https://github.com/linebender/peniko/pull/26) - Add `serde` feature to enable serde support by [@ratmice](https://github.com/ratmice)
+- `serde` feature to enable serde support ([#26] by [@ratmice][])
+
+## [0.1.0][] (2024-02-15)
 
-## 0.1.0 (2024-02-15)
+This release has an [MSRV] of 1.70.
 
 - Initial release
+
+[MSRV]: README.md#minimum-supported-rust-version-msrv
+
+[#26]: https://github.com/linebender/peniko/pull/26
+
+[@ratmice]: https://github.com/ratmice
+[@DJMcNab]: https://github.com/DJMcNab
+[#40]: https://github.com/linebender/peniko/pull/40
+
+[Unreleased]: https://github.com/linebender/peniko/compare/v0.1.1...HEAD
+[0.1.1]: https://github.com/linebender/peniko/compare/v0.1.0...v0.1.1
+[0.1.0]: https://github.com/linebender/peniko/releases/tag/v0.1.0
```

Comment:
Is this mostly to be able to use that trick for docs.rs? Or for readability?

Reply:
Can you clarify your question? This is just mindlessly applying the pattern of linebender/vello#604.

The changelog isn't included in rustdoc (at least not currently)

Reply:
I mean link references vs in place links, but I see that this is just the general format there